addons/mass_mailing/report/mailing_trace_report_views.xml

```diff
@@ -32,6 +32,8 @@
             <field name="model">mailing.trace.report</field>
             <field name="arch" type="xml">
                 <search string="Mass Mailing Statistics">
+                    <field name="name" string="Mailing" filter_domain="[('name', 'ilike', self)]"/>
```

Comment:
What is the use of `filter_domain="[('name', 'ilike', self)]"` ?
The search on "name" should do that by default, no need to specify it.

Reply:
Done